### PR TITLE
Update PKGBUILD for Arch linux

### PR DESCRIPTION
**Pull request by kdehairy**
- Depend on systemd in core repo (python-systemd)
- Add python-setuptools to guild dependencies
- Increment txdbus version to 1.1.0
- Get txdbus tar from Pypi repo instead of github
- Some cleanups in the install script

### CODE REVIEW

**Comment by tkluck** [comment]
Hey @kdehairy , this looks really good to me. I just added a `Dockerfile` to the archlinux directory that I used to test these changes.

Do I understand correctly that this new version of `PKGBUILD` will not work until we tag a release that contains #24 ?

I'll approve this request; if you approve of the commit I added, feel free to merge it.

**Comment by kdehairy** [comment]
hey @tkluck I modified the Dockerfile to run a fully working installation of the package (as a Systemd service). This will enable us to use it to test the package exactly as it is going to run on the users' machines.

I ran into merge conflicts with your latest changes (using the `--bind 0.0.0.0`). I commented it (not to loose the change) until we decide which approach we like the most. Let me know what you think

File: test/test-arch-package.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+if [[ "x${SRC_DIR}" == "x" ]]; then
+	echo "Usage:"
+	echo "  test-arch-package <project_src_dir>"
+	exit 1
+fi
+
+cd $SRC_DIR
+
+sudo docker build -t pac4cli -f archlinux/Dockerfile .
```
Comment (tkluck) [inline test/test-arch-package.sh]: On debian, it's possible to do `addgroup $USER docker` and run `docker` without `sudo` after that. I have a mild preference for not having `sudo` in the script, but I won't insist if that makes it unergonomic on your setup. What do you think?

**Comment by tkluck** [comment]
I like your approach a lot @kdehairy . It makes a lot of sense to test the end-to-end build, install, and running of the daemon under systemd. If we pull it off for archlinux, I'll try the same for debian.

As we discussed in person today, it's quite hard to figure out what the test script should do -- docker isn't super conducive to running several processes inside the same container, and after solving that, you run into the `init` daemon being picky about whether there's a login session.

How about we merge it as-is for now (modulo your resolution of my `sudo` comment)? That'll give me some proper way to test the archlinux stuff from now on, even if I'll have to do some of it through manual inspection of the container.

File: test/test-arch-package.sh (hunk 1 above)
Comment (tkluck) [reply test/test-arch-package.sh]: Oh, and can we put `archlinux` somewhere in the tag for the image?

Comment (kdehairy) [reply test/test-arch-package.sh]: Make sense. I merged the branch for now (not to block others) and will do the requested changes soon

**Pull request merged**